README.md

```diff
@@ -35,52 +42,79 @@ Windows](https://developer.concordium.software/en/mainnet/net/installation/downl
+    - Use the `x86_64-pc-windows-gnu` toolchain by choosing it during
        installation or by running `rustup toolchain default
        stable-x86_64-pc-windows-gnu`.
-   * *Recommended after installing: Set the default Rust version to 1.53 by running `rustup default 1.53`*.
+  - *Recommended after installing: Set the default Rust version to 1.53 by running `rustup default 1.53`*.
 
-* Install the [protoc](https://github.com/google/proto-lens/blob/master/docs/installing-protoc.md) tool for generating protobuf files:
-   * MacOS: `brew install protobuf`
-   * Ubuntu 19.10: `sudo apt install protobuf-compiler`
-   * Windows: `stack exec -- pacman -S mingw-w64-x86_64-protobuf`
+- Install the [protoc](https://github.com/google/proto-lens/blob/master/docs/installing-protoc.md) tool for generating protobuf files:
+  - MacOS: `brew install protobuf`
+  - Ubuntu 19.10: `sudo apt install protobuf-compiler`
+  - Windows: `stack exec -- pacman -S mingw-w64-x86_64-protobuf`
 
-* Install development libraries for PostgreSQL:
-   * MacOS: `brew install postgresql`
-   * Ubuntu 19.10: `sudo apt install libpq-dev postgresql-common` 
-   * Windows: `stack exec -- pacman -S mingw-w64-x86_64-postgresql`
+- Install development libraries for PostgreSQL:
+  - MacOS: `brew install postgresql`
+  - Ubuntu 19.10: `sudo apt install libpq-dev postgresql-common`
+  - Windows: `stack exec -- pacman -S mingw-w64-x86_64-postgresql`
 
-* Initialize submodule dependencies after cloning (`git submodule update --init --recursive`).
+- Initialize submodule dependencies after cloning (`git submodule update --init --recursive`).
 
 ## Build
 
 ``` sh
 stack build
 ```
 
+### MacOS Specific
+
+You may need to add the following entries to your `~/.stack/config.yaml` for the libraries installed via `brew`:
+
+```yaml
+extra-lib-dirs:
+- /opt/homebrew/lib
+
+extra-include-dirs:
+- /opt/homebrew/include/
+```
+
+### M1 MacOS Specific
+
+You may need to add the following entry to your `~/.stack/config.yaml` for the `libffi` include:
+
+```yaml
+extra-include-dirs:
+- /Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/ffi/
+```
+
+To determine the exact path to the `libffi` include directory, run the following command:
+
+```sh
+pkg-config --cflags libffi
+```
+
 ## Usage
 
 Run using `stack run concordium-client -- [BACKEND] COMMAND [ARGS...]`, where
 
-* `BACKEND` is the GRPC server on which to perform the actions/queries.
+- `BACKEND` is the GRPC server on which to perform the actions/queries.
   It's specified using the flags `--grpc-ip`, `--grpc-port`, and `--grpc-target`
   (might be needed when calling through proxy like, say, on the testnet).
 
-* `COMMAND` is a command from one of the categories described below in [Commands](#commands).
+- `COMMAND` is a command from one of the categories described below in [Commands](#commands).
 
-* `ARGS` is the list of arguments provided to `COMMAND`.
+- `ARGS` is the list of arguments provided to `COMMAND`.
```

Comment:
None of this was necessary.

Reply:
It's about consistency, something markdown linter complains about.